build-site.sh

```diff
@@ -73,10 +73,6 @@ check_environment_variables() {
         echo -e "${RED}JEKYLL_ENV needs to be set${NC}"
         exit 1
     fi
-    if [ "$JEKYLL_ENV" != "staging" ] && [ "$JEKYLL_ENV" != "production" ]; then
```

Comment:
Are you just letting `JEKYLL_ENV` fall through from the website's `build-site.sh` script? Is there a corresponding documentation update, or will that be delayed until the documentation is consolidated?

Reply:
Are you removing `check_environment_variables` below?

Reply:
I'm still testing that `JEKYLL_ENV` is set - I'm just not enforcing the permitted values. For now, I'm intending to leave the documentation as stating that it must be set to `staging` or `production`.